src/types/audio-plugin.ts

```diff
@@ -0,0 +1,3 @@
+import {IconDto} from '@playkit-js/ui-managers';
+
+export type AudioPluginDto = Omit<IconDto, 'ariaLabel' | 'order' | 'component' | 'presets'>;
```

Comment:
This file only creates audio plugin dto, so first please rename the as it is not a main file for audio-plugin but only a a dto file.
Now regarding the name of the dto class, if it is inheriting IconDto, then a proper name will be AudioIconDto, please rename class and file

Reply:
yes but it is not an audio icon it is an audio plugin icon, IMO the former needs to be fixed not the latter (to be UpperBarIconDto) otherwise the meaning will be an icon of the audio player

Reply:
done